src/components/date-time-picker/day-view/day-view.component.html

```diff
@@ -8,12 +8,13 @@
     <tbody role="grid">
         <tr role="row" *ngFor="let row of dayService.grid$ | async; trackBy: trackWeekByFn">
 
-            <td *ngFor="let item of row; trackBy: trackDayByFn" class="date-cell" role="gridcell">
+            <td *ngFor="let item of row; trackBy: trackDayByFn" class="date-cell">
 
                 <button type="button"
                         uxFocusIndicator
                         uxFocusIndicatorOrigin
                         class="date-button"
+                        role="gridcell"
```

Comment:
I'm seeing more errors out of the axe tool with this placement, so I think the path of least resistance is to remove `role="grid"`, `role="row"`, and `role="gridcell"` altogether.